actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -116,12 +116,12 @@ def use_cookies_with_metadata
   #     cookies[:login] = { value: "XJ-122", expires: Time.utc(2020, 10, 15, 5) }
   #
   #     # Sets a signed cookie, which prevents users from tampering with its value.
-  #     # It can be read using the signed method `cookies.signed[:name]`
+  #     # It can be read using the signed method `cookies.signed[:user_id]`
```

Comment:
Maybe move this line after and show what it returns?
```suggestion
  #     # It can be read using the signed method.
  #     cookies.signed[:user_id] # => 123
```